#goal definition
geometry_msgs/PoseStamped plan_to
---
#result definition
bool got_plan
---
#feedback
geometry_msgs/Pose plan
